src/__tests__/integration.test.tsx

```diff
@@ -40,7 +40,7 @@ const createApp = ({
     ssr,
   });
 
-  LooselyLazy.init(hydrate ? MODE.HYDRATE : MODE.RENDER);
+  LooselyLazy.init({ mode: hydrate ? MODE.HYDRATE : MODE.RENDER });
```

Comment:
Moved `manifest` from being an argument of `getAssetUrls` to be one of `init`, that way we can use the manifest across all files instead of just to get the assets urls. 
This is an API breaking change but should be easy to migrate